CASC/PortfolioMode.cpp

```diff
@@ -45,15 +45,17 @@
 
 #include "PortfolioMode.hpp"
 
-using namespace std;
 using namespace Lib;
 using namespace CASC;
 using Lib::Sys::Multiprocessing;
+using std::cout;
+using std::cerr;
+using std::endl;
 
```

Comment:
What do you think, @MichaelRawson , about this particular file now, after my change?

Reply:
Funnily, `move` worked without the full qualification even after I removed `using namespace std;`.

Reply:
Commit message probably a little harsh! But looks good to me.

Reply:
> Funnily, `move` worked without the full qualification even after I removed `using namespace std;`.

Could be! I assume this is ADL at work.